recipes/parsplice/build.sh

```diff
@@ -0,0 +1,3 @@
+mkdir build
+cd build
+cmake -DCMAKE_PREFIX_PATH="${PREFIX}"  -DCMAKE_INSTALL_PREFIX="${PREFIX}" ../ 
```

Comment:
Did you forget to run ``make``?
```suggestion
cmake -DCMAKE_PREFIX_PATH="${PREFIX}"  -DCMAKE_INSTALL_PREFIX="${PREFIX}" ../ 
make -j${CPU_COUNT}
```